tests/vars.sample.yaml

```diff
@@ -36,6 +46,15 @@ source_ovndb_ip: 192.168.122.100 #CUSTOMIZE_THIS
 # EDPM node IP
 edpm_node_ip: 192.168.122.100 #CUSTOMIZE_THIS
 
```

Comment:
do we want or need these vars. i am planning to drop them, but we have an interface/documentation issue we need to clarify.

Reply:
As discussed on Monday, IMO let's drop these. They aren't used anywhere in code.

Reply:
ok i am rebasing and will drop these